src/components/form-landing/form-landing.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { Vedtak } from '../../types/vedtak';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkAuth } from '../../store/actions';
+import { Store } from '../../store/reducer';
+import WithLoading from '../../components/general/loading/withLoading';
+import { logInfo } from '../../utils/logger/frontendLogger';
+import { validVedtakQuery, elementAsVedtak } from '../../mock-api/get/vedtak';
+import MainFormPage from '../../pages/form-landing-page/main-form-page';
+import Error from '../../components/error/error';
+
+const FormLanding = (props: any) => {
+    const dispatch = useDispatch();
+    const { loading, chosenYtelse } = useSelector((state: Store) => state);
+
+    const [chosenVedtak, setChosenVedtak] = useState<Vedtak>();
+    const [temaNotSet, setTemaNotSet] = useState<boolean>(false);
+
+    useEffect(() => {
+        dispatch(checkAuth());
+    }, [dispatch]);
+
+    useEffect(() => {
+        if (validVedtakQuery(props.query)) {
+            setChosenVedtak(elementAsVedtak(props.query));
+        } else {
+            setTemaNotSet(true);
+        }
+    }, [dispatch, props.query]);
+
+    logInfo('Form landing page visited.', { chosenYtelse: chosenYtelse, referrer: document.referrer });
+
+    if (temaNotSet) {
+        return (
+            <Error
+                error={{ code: 400, text: 'Tema er ikke spesifisert, vennligst gå gjennom våre innsynsløsninger.' }}
```

Comment:
Jeg tror ikke ordet innsynsløsninger vil gi mening for bruker. I tillegg er tema et domeneord som bruker ikke forstår. Foreslår: 
Ytelse du ønsker å klage på er ikke spesifisert. Dersom du navigerer til denne siden via en lenke på Ditt NAV vil ytelse bli satt riktig.

I tillegg: Bør vi logge denne til frontendlogger med document.referrer?